aea/connections/base.py

```diff
@@ -77,7 +77,7 @@ def __init__(
 
         :param configuration: the connection configuration.
         :param identity: the identity object held by the agent.
-        :param cryptos: the crypto store for encrypted communication.
+        :param crypto_store: the crypto store for encrypted communication.
```

Comment:
cryptos is confusing; as the store itself has `crypto_objects`. We should be explicit!